eZ/Publish/Core/Repository/RoleService.php

```diff
@@ -1410,7 +1382,6 @@ protected function validatePolicy($module, $function, array $limitations)
                         "'{$limitation->getIdentifier()}' was found several times among the limitations"
                     );
                 }
-
```

Comment:
can keep this line

Reply:
Sorry, it was deleted by mistake. I added it back 🙂